code/Makefile

```diff
@@ -0,0 +1,23 @@
+install:
+	npm i @informalsystems/quint -g
+	cargo install --locked cargo-nextest
+
+lint:
+	cargo fmt --all --check
+	cargo clippy --workspace --all-features --all-targets -- -D warnings
+
+lint-fix:
+	cargo fmt --all
+	cargo clippy --fix --allow-dirty --allow-staged --workspace --all-features --all-targets -- -D warnings
```

Comment:
Let's switch these two steps, as fixing clippy lints automatically sometimes messes up the formatting